alfresco-health-processor-platform/src/main/java/eu/xenit/alfresco/healthprocessor/plugins/api/ToggleableHealthProcessorPlugin.java

```diff
@@ -0,0 +1,28 @@
+package eu.xenit.alfresco.healthprocessor.plugins.api;
+
+import eu.xenit.alfresco.healthprocessor.reporter.api.NodeHealthReport;
+import java.util.Set;
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+import org.alfresco.service.cmr.repository.NodeRef;
+import org.slf4j.Logger;
+
+@Slf4j
+@Data
```

Comment:
Is it `@Data` though? I think you only need a getter & setter for `enabled`.
We don't need ` @ToString` or `@EqualsAndHashCode` or `@RequiredArgsConstructor`?

Reply:
One could argue that one annotation is better than 2, but it's abuse of the data abstraction :smile:

Reply:
SonarCube complains that I should add overwrite `equals(...)` in these classes :upside_down_face: